code/ship/ship.cpp

```diff
@@ -7332,6 +7332,9 @@ void ship_render_cockpit(object *objp)
 	vm_vec_unrotate(&pos, &sip->cockpit_offset, &eye_ori);
 
 	bool shadow_override_backup = Shadow_override;
+	float fov_backup = Proj_fov;
+
+	g3_set_fov(Sexp_fov <= 0.0f ? COCKPIT_ZOOM_DEFAULT : Sexp_fov);
```

Comment:
Am I reading this right that the sexp would adjust both the gobal and cockpit fov?

Reply:
Ok.  I guess it can always be expanded with a cockpit fov sexp later, especially if it's requested.